### ISSUE TITLE
Two input field should not be empty while generating table 

### ISSUE THREAD

**Issue by Siddhesh172004** [LunarMarathon/graphToTable#33]
![image](https://user-images.githubusercontent.com/84581657/195639414-18b177e4-0b0d-4a66-88ae-2c28fe6e4c33.png)

In the above Image I think there should be a condition  given that when any one  of the two input are kept empty then it should give a alert message. As we  see in the above image the input field when are kept empty and we generate table then it puts empty columns. So I feel this needs to be fixed. Let me know what you think about it. 

Kindly assign me this issue.

**Comment by LunarMarathon** [comment]
There's no form so required can't be used. Will the alert be styled or default?

**Comment by Siddhesh172004** [comment]
I was thinking a styled alert. Which one would you prefer? Styled or default let me know so I can contribute to it.

**Comment by LunarMarathon** [comment]
A styled one please! I've made UI changes so please use the new color palette in the CSS file. The search button needs it too btw but only checking for the first i/p is enough. The second i/p isn't require for search. Let me know if you have a rough design (or else it's a headache while reviewing) Thanks!

**Comment by Siddhesh172004** [comment]
![image](https://user-images.githubusercontent.com/84581657/196004352-e40884f0-a7eb-434f-9c1d-83fb77d4bd91.png)


I have made a styled alert when any one of the input empty it will give a alert when user wants to createtable.

![image](https://user-images.githubusercontent.com/84581657/196004398-5b889aef-8a00-405f-b745-43693332af6a.png)

 I have also added dark mode to it . We can change the color let me know if its okay or not.

Kindly assign me this issue so i can send  PR.
